stress-example/run_image.sh

```diff
@@ -0,0 +1,15 @@
+set -x
+
+if [ -z "$1" ]
+then
+  echo "No time (seconds) for running supplied. Use e.g. 'bash run_image.sh 60'" && exit 1
+fi
+
+bash ./env_vars_config.sh && \
```

Comment:
let's have it one step at a time with some prints in between? Also we need to prepare different `query.yml` for each `type of indexer` (Faiss and Annoy)

Reply:
What do you mean, "one step at a time"? Wasn't the intention of the script to run the indexing and querying in one command?

Reply:
Yes, I need to figure out how to best do that. Separate `query.yml`-s could be the way